pegasus-spark-common/src/main/java/com/xiaomi/infra/pegasus/spark/CommonConfig.java

```diff
@@ -0,0 +1,31 @@
+package com.xiaomi.infra.pegasus.spark;
+
+import java.io.Serializable;
+
+/** The config class contains the common config for coldBackupConfig and bulkLoadConfig */
+public class CommonConfig implements Serializable {
+
+  public RemoteFileSystem remoteFileSystem;
+
+  public String remoteFileSystemURL;
+  public String remoteFileSystemPort;
+  public String clusterName;
+  public String tableName;
```

Comment:
make them `protected`, since they are used internally for coldBackupConfig and bulkLoadConfig